jenkins/ci.suse.de/openstack-ardana.yaml

```diff
@@ -332,6 +338,12 @@
                   update.yml
           fi
 
+          # Deploy CaaSP
+          if [ -n "$want_caasp" ]; then
+            ansible-playbook -v -i hosts \
```

Comment:
I am not familiar enough with the product or the roadmap, but couldn't this sort of play be part of the product instead of part of the gating? The less code in gating, the easier to maintain in the long run.

Reply:
This is something to consider I guess, we could have a playbook do the caasp deployment in our product.

Reply:
This is something to consider I guess, we could have a playbook do the caasp deployment in our product.

Reply:
I think it's worth researching, but that shouldn't prevent this to merge in the first place.